### PR TITLE
Checkbox

### PR DESCRIPTION
**Pull request by alexbchr**

### CODE REVIEW

File: src/components/forms/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      borderRadius: theme.radius.small,
+      width: size,
+      height: size,
+      marginVertical: (theme.font.size.regular.height - size) / 2,
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    backgroundChecked: {
+      borderColor: theme.colors.fill.accent.default,
+    },
```
Comment (hansgx) [inline src/components/forms/Checkbox/Checkbox.tsx:48]: Same [comment](https://github.com/hansgx/Cubes/pull/13#discussion_r445073188) as the one from #13

File: src/components/forms/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  checked: boolean
+  /**
+   * Called when selection state changes. Should propagate change to `checked` prop.
+   */
+  onChange: (checked: boolean) => void
+  /**
+   * Additional text to aid in use.
+   */
+  helpText?: string
+}
```
Comment (hansgx) [inline src/components/forms/Checkbox/Checkbox.tsx:26]: See [this comment](https://github.com/hansgx/Cubes/pull/13#discussion_r445074065)

File: src/components/forms/Checkbox/Checkbox.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  }))
+
+  const onClick = useCallback(() => onChange(!checked), [checked, onChange])
+
+  return (
+    <TouchableWithoutFeedback onPress={onClick}>
+      <View>
+        <Box horizontal space="medium" paddingY="medium">
+          <View style={[styles.background, checked ? styles.backgroundChecked : null]}>
+            {checked ? <Icon name="clear" color="accent" /> : null}
```
Comment (hansgx) [inline src/components/forms/Checkbox/Checkbox.tsx:58]:
```suggestion
            {checked && <Icon name="clear" color="accent" />}
```

File: src/components/forms/Checkbox/Checkbox.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+export const Basic: StoryFn<CheckboxProps> = props => {
+  const [checked, setChecked] = useState(props.checked)
+
+  return <Checkbox {...props} checked={checked} onChange={setChecked} />
+}
+
+Basic.args = {
+  checked: false,
+  label: 'A smoothie',
+  helpText: 'With orange juice, mangos and love ❤️',
```
Comment (hansgx) [inline src/components/forms/Checkbox/Checkbox.stories.tsx:21]: Maybe change the sample text to be different from `RadioButton` (to make it instantly obvious for people quickly scanning the docs)

Comment (alexbchr) [reply src/components/forms/Checkbox/Checkbox.stories.tsx]: Done :wink:

File: src/components/forms/Checkbox/Checkbox.tsx (hunk 2 above)
Comment (alexbchr) [reply src/components/forms/Checkbox/Checkbox.tsx]: Done :wink:

**Pull request merged**